packages/react-textarea/src/stories/TextAreaUncontrolled.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { TextArea, TextAreaProps } from '../index';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles, shorthands } from '@griffel/react';
+
+const useStyles = makeStyles({
```

Comment:
Nit: In writing all the stories for SpinButton I found myself repeating styles like this in every story and I settled on adding a decorator in the root .stories file that imports all the other stories and applying styles there. I like this because the styling isn't really critical to the individual stories and it clears up the stories to emphasize the topic at hand.

Reply:
The problem with that is the decorator doesn't get shown with "show code" or exported to codepen, so someone trying to use the example may be confused when they copy the code and it doesn't look that way in their app (due to missing magical mystery styles).

Reply:
^^^ this, unfortunately our codesandbox export only works for one single module, if you use a decorator it will make the actual exported examples look quite funny. This already happens for certain examples AFAR